com.yacreader.YACReader.yml

```diff
@@ -0,0 +1,127 @@
+    modules:
+      - name: unarr
+        buildsystem: cmake-ninja
+        sources:
+          - type: git
+            url: https://github.com/selmf/unarr.git
+            commit: cf4deedfdd2b815ae2ad0f24cfda72c3d8083d14
+      - name: pdfium
+        buildsystem: simple
+        build-commands:
+          # https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=libpdfium-nojs
+
+          # Patch abseil build to be static
+          - sed -i -e 's/component(/static_library(/' -e 's/is_component_build(/false/' pdfium/third_party/abseil-cpp/BUILD.gn
+          # Use system provided icu library (unbundling)
+          - mkdir -p pdfium/third_party/icu
+          - ln -sf ../../build/linux/unbundle/icu.gn pdfium/third_party/icu/BUILD.gn
+          # Decode shim header script needed to unbundle icu (gittiles is weird)
+          - mkdir -p pdfium/tools/generate_shim_headers
+          - base64 --decode generate_shim_headers.py > pdfium/tools/generate_shim_headers/generate_shim_headers.py
+          # Create fake gclient_args.gni file to satisfy include list for build/config/compiler/compiler.gni
+          - touch pdfium/build/config/gclient_args.gni
+          # Exclude test fonts from build
+          - sed -i '/"\/\/third_party\/test_fonts",/s/^/#/g' pdfium/testing/BUILD.gn
+          # Remove cross-compile-toolchain-prefix for gcc (needed for aarch64 build)
+          - sed -i 's/"aarch64-linux-gnu-"/""/g' pdfium/build/toolchain/linux/BUILD.gn
+          # Build
+          - gn --root=pdfium gen pdfium/out/Release --args="is_clang=false use_sysroot=false is_debug=false symbol_level=0 pdf_enable_v8=false pdf_enable_xfa=false treat_warnings_as_errors=false use_system_libjpeg=true use_system_zlib=true pdf_bundle_freetype=false use_system_freetype=true use_system_lcms2=true use_system_libpng=true use_custom_libcxx=false pdf_is_standalone=true use_system_libopenjpeg2=true is_component_build=true use_gold=false"
+          - ninja -C pdfium/out/Release pdfium
+          # TODO: use a variable for version
+          - sed -e 's/@VERSION@/5249/g' -e 's#^\s*prefix\s*=.*$#prefix='"$FLATPAK_DEST"'#' -i libpdfium.pc
+          # Install
+          - install -D -m644 pdfium/LICENSE "$FLATPAK_DEST/share/licenses/pdfium/LICENSE"
+          - install -D pdfium/public/*.h --target-directory="$FLATPAK_DEST/include/pdfium"
+          - install -D pdfium/public/cpp/* --target-directory="$FLATPAK_DEST/include/pdfium/cpp"
+          - install -D pdfium/docs/* --target-directory="$FLATPAK_DEST/share/doc/pdfium"
+          - install -Dm755 pdfium/out/Release/libpdfium.so --target-directory="$FLATPAK_DEST/lib"
+          - install -Dm644 libpdfium.pc --target-directory="$FLATPAK_DEST/lib/pkgconfig"
+        sources:
+          - type: git
+            url: https://pdfium.googlesource.com/pdfium
+            # curl 'https://omahaproxy.appspot.com/linux?channel=stable' | cut -d'.' -f 3
+            branch: chromium/5249
```

Comment:
The problem is precisely this – there's zero guarantee this branch points to the same commit it has been during the build. It may affect reproducibility, should someone want to test whether Flathub build is legit. Wouldn't leaving a comment about which branch was used to find the commit be sufficient?